packages/stryker/src/input/InputFileResolver.ts

```diff
@@ -0,0 +1,174 @@
+import * as path from 'path';
```

Comment:
This file should be looked at, contains more than one class and is hard to understand.